src/index.html

```diff
@@ -1,14 +1,100 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="https://marksheet.io/css/reset.css">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <header class="head">
+      <div class="logo">
+        <a href="#home">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+      </div>
+      <nav class="nav__menu">
+        <ul class="ul-items">
```

Comment:
```suggestion
        <ul class="nav__list">
```
It will be better name